executor/join.go

```diff
@@ -201,7 +201,7 @@ func (e *HashJoinExec) fetchOuterChunks(ctx context.Context) {
 			return
 		}
 		var outerResource *outerChkResource
-		ok := true
+		var ok bool
```

Comment:
The value of `ok` is true in the original code, why change it?

Reply:
Indeed , it has not used at all.
Next time it is used by `case outerResource, ok = <-e.outerChkResourceCh:`
It has been overwritten.